src/coreComponents/linearAlgebra/interfaces/hypre/HypreInterface.cpp

```diff
@@ -42,6 +42,14 @@ void HypreInterface::initialize()
   hypre_HandleSpgemmUseVendor( hypre_handle() ) = 0;
 #endif
   HYPRE_SetMemoryLocation( hypre::memoryLocation );
+
+  // Hypre version info
+#if defined(HYPRE_DEVELOP_STRING) && defined(HYPRE_BRANCH_NAME)
+  GEOSX_LOG_RANK_0( "  Using hypre development version: " << HYPRE_DEVELOP_STRING <<
+                    " (" << HYPRE_BRANCH_NAME << ")" );
+#elif defined(HYPRE_RELEASE_VERSION)
+  GEOSX_LOG_RANK_0( "  Using hypre release version: " << HYPRE_RELEASE_VERSION );
+#endif
```

Comment:
if either `HYPRE_DEVELOP_STRING` , and `HYPRE_BRANCH_NAME` is not defined, this will pass through without any output.

Reply:
We always have `HYPRE_RELEASE_VERSION` defined from hypre, so there will be an output from line 51. I could have replaced line 50 with just an `#else` but decided to do `#elif defined(HYPRE_RELEASE_VERSION)`